package-lock.json

```diff
@@ -15721,9 +15721,9 @@
             }
         },
         "node_modules/@types/css-modules": {
-            "version": "1.0.4",
-            "resolved": "https://registry.npmjs.org/@types/css-modules/-/css-modules-1.0.4.tgz",
-            "integrity": "sha512-ZDHVt5uCphHH9OY7eWqI5b7jlmAcpyDdEjv2R8+uARv/nt7A7KZM9429s1pRGZGCbAGhgzqxr4DI/YGqJR9keA=="
+            "version": "1.0.3",
+            "resolved": "https://registry.npmjs.org/@types/css-modules/-/css-modules-1.0.3.tgz",
+            "integrity": "sha512-dFJKRbgpu3sMsiDYX7dSL7ZMD/9q1UxbU/dZfIuVsIyRsST/agUNfR24biGeLOmh3YcdUmj71TadJ0mssFVNfA=="
```

Comment:
Why do we need a downgrade for this?

Reply:
I had a conflict in this file when I pulled from master. I deleted the file and ran `npm run bootstrap:dev`. I'm not sure why this happpened. Let me run it again